src/app.py

```diff
@@ -73,24 +73,32 @@ def process_workflow_job():
 
     elif action == "in_progress":
         job_requested = jobs.get(job_id)
+        time_to_start = None
         if not job_requested:
-            app.logger.warning(f"Job {job_id} is {action} but not stored!")
-            time_to_start = 0
+            app.logger.error(f"Job {job_id} is {action} but not stored!")
```

Comment:
Keep `warning` for **non-fatal** errors. (Anything as ERROR is candidate to generate an alarm).

```suggestion
            app.logger.warning(f"Job {job_id} is {action} but not stored!")
```